packages/backend/src/app/workers/sandbox/isolate-sandbox.ts

```diff
@@ -61,6 +61,7 @@ export class IsolateSandbox extends AbstractSandbox {
                 '--env=HOME=/tmp/',
                 '--env=NODE_OPTIONS=\'--enable-source-maps\'',
                 `--env=AP_PIECES_SOURCE=${pieceSource}`,
+                '--env=AP_CODE_EXECUTOR_SANDBOX_TYPE',
```

Comment:
Nit: pass the raw value in case we have default for that

Reply:
fixed in 7845ba1